<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminDialog</class>
 <widget class="QDialog" name="adminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>395</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>395</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ethernet_Mode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>201</width>
     <height>46</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="internet_Mode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>46</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>100</y>
     <width>106</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>internet  Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>180</y>
     <width>106</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>ethernet Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>145</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nineButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>225</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>385</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>305</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;----</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eightButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>225</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>305</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sevenButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>225</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sixButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>145</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>65</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>65</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>65</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>145</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="subnetLabel">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>260</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Subnet Mask</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ipTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="tabStopWidth">
    <number>13</number>
   </property>
   <property name="placeholderText">
    <string>|||||||||||||||||</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="subnetTextEdit">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>280</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>||||||||||||||||</string>
   </property>
  </widget>
  <widget class="QPushButton" name="periodButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>305</y>
     <width>89</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>10</y>
     <width>66</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="gatewayTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>||||||||||||||||</string>
   </property>
  </widget>
  <widget class="QLabel" name="gatewayLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Gateway</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="dnsTextEdit">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>340</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>||||||||||||||||</string>
   </property>
  </widget>
  <widget class="QLabel" name="dnsLabel">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>320</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Primary DNS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dnsLockbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>86</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>DNS Lock</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
